src/main/java/com/cloudbees/plugins/credentials/CredentialsProviderManager.java

```diff
@@ -137,6 +146,17 @@ public static CredentialsProviderManager getInstanceOrDie() {
         return instance;
     }
 
+    /**
+     * Helper method to know if the fingerprints are processed
+     *
+     * @return if the fingerprints are processed
+     * @since TODO
+     */
+    public static boolean isFingerprintEnabledOrDefault() {
```

Comment:
Seems a little longer than needed. Especially the OrDefault. That probably folds into the enabled case.

Reply:
Method removed due to the system property 👍